cfgov/unprocessed/css/video-player.less

```diff
@@ -85,8 +94,17 @@
     }
 
     &__bottomRight {
-        bottom: @gutter-half__em;
-        right: @gutter-half__em;
+        right: unit( @grid_gutter-width / 2 / ( @play-btn-icon-size * @base-font-size-px ), em );
+        bottom: unit( @grid_gutter-width / 2 / ( @play-btn-icon-size * @base-font-size-px ), em );
```

Comment:
Calculated sizes are getting complex. I wonder if we should document how to calculate these somewhere in the CF docs.

Reply:
That's a good idea. Maybe we should have a document of "common idioms or patterns in our Less".